study.md

```diff
@@ -19,38 +19,42 @@ Please write the code that would take the following integer `007` and return the
 string `"700"`
 
 ```ruby
-  # your answer here
+007.to_s.reverse
 ```
 
 Please write the code that takes the following array `[23,56,3,7]` sorts it
 then reverses that array returning and modifying the refrence to the original
 array.
 
 ```ruby
-  # your answer here
+[23,56,3,7].sort.reverse!
 ```
 
 What does an exclamation point after a method usually signify?
 
 ```ruby
-  # your answer here
+  modifies the original data rather than making a copy
 ```
 
 How do you create a class in Ruby?
 
 ```ruby
-  # your answer here
+class Name
+
+end
 ```
 
 If I modify a class in Ruby will already created instances of that class have
 the methods I added?
 
 ```ruby
-  # your answer here
+  Yes. All new objects created will have but it is also available in existing objects of that class.
 ```
 
 Please give an example of iterating in Ruby?
 
 ```ruby
-  # your answer here
+  @method.each do |thing|
+    puts "block"
```

Comment:
you didn't actually use "thing" there.  "thing" would be the current iteration much like [i] in a javascript loop